pygrouper/client.py

```diff
@@ -2,16 +2,19 @@
 """
 
 import requests
+from .exceptions import GrouperAPIException, GrouperAPIRequestsException, GrouperAPIError
 
 WS_VERSIONS = [
     'v2_2_000'
 ]
```

Comment:
Can we add newer versions to this array?  At least v2_5_000 would be nice.  Let's keep the default as-is, though, to avoid any breakage at this time.